pkg/standalone-utils/test/foundry/BalancerFeeBurner.t.sol

```diff
@@ -217,6 +226,184 @@ contract BalancerFeeBurnerTest is BaseVaultTest {
+        vm.prank(alice);
+        IERC20(address(waDAI)).transfer(address(feeSweeper), TEST_BURN_AMOUNT);
+
+        IBalancerFeeBurner.SwapPathStep[] memory steps = new IBalancerFeeBurner.SwapPathStep[](1);
+        steps[0] = IBalancerFeeBurner.SwapPathStep({ pool: address(waDAI), tokenOut: dai, isBuffer: true });
+
+        Balances memory balancesBefore = getBalances();
+
+        vm.prank(alice);
+        feeBurner.setBurnPath(waDAI, steps);
+
+        uint256 snapshot = vm.snapshot();
+        uint256 amountOut = _vaultPreviewRedeem(waDAI, TEST_BURN_AMOUNT);
+        vm.revertTo(snapshot);
+
+        vm.startPrank(address(feeSweeper));
+        IERC20(address(waDAI)).forceApprove(address(feeBurner), TEST_BURN_AMOUNT);
+
+        feeBurner.burn(address(0), waDAI, TEST_BURN_AMOUNT, dai, amountOut, alice, orderDeadline);
+        vm.stopPrank();
+
+        Balances memory balancesAfter = getBalances();
+
+        assertEq(
+            balancesAfter.userTokens[waDaiIdx],
+            balancesBefore.userTokens[waDaiIdx] - TEST_BURN_AMOUNT,
+            "waDAI balance should decrease (feeSweeper)"
+        );
+        assertEq(
+            balancesAfter.aliceTokens[daiIdx],
+            balancesBefore.aliceTokens[daiIdx] + amountOut,
+            "DAI balance should increase (alice)"
+        );
+        assertEq(
+            balancesAfter.vaultTokens[waDaiIdx],
+            balancesBefore.vaultTokens[waDaiIdx] + TEST_BURN_AMOUNT,
+            "waDAI balance should increase (vault)"
+        );
+        assertEq(
+            balancesAfter.vaultTokens[daiIdx],
+            balancesBefore.vaultTokens[daiIdx] - amountOut,
+            "DAI balance should decrease (vault)"
+        );
+
+        assertEq(waDAI.balanceOf(address(feeBurner)), 0, "FeeBurner should not hold waDAI");
+        assertEq(dai.balanceOf(address(feeBurner)), 0, "FeeBurner should not hold DAI");
+    }
+
+    function testBurnWrapUnwrap() external {
+        vm.prank(alice);
+        IERC20(address(waDAI)).transfer(address(feeSweeper), TEST_BURN_AMOUNT);
+
+        IBalancerFeeBurner.SwapPathStep[] memory steps = new IBalancerFeeBurner.SwapPathStep[](2);
+        steps[0] = IBalancerFeeBurner.SwapPathStep({ pool: address(waDAI), tokenOut: dai, isBuffer: true });
+        steps[1] = IBalancerFeeBurner.SwapPathStep({ pool: address(waDAI), tokenOut: waDAI, isBuffer: true });
+
+        Balances memory balancesBefore = getBalances();
+
+        vm.prank(alice);
+        feeBurner.setBurnPath(waDAI, steps);
+
+        uint256 snapshot = vm.snapshot();
+        uint256 amountOut = _vaultPreviewRedeem(waDAI, TEST_BURN_AMOUNT);
+        vm.revertTo(snapshot);
+
+        vm.startPrank(address(feeSweeper));
+        IERC20(address(waDAI)).forceApprove(address(feeBurner), TEST_BURN_AMOUNT);
+
+        uint256 minAmountOut = TEST_BURN_AMOUNT - DELTA;
+        feeBurner.burn(address(0), waDAI, TEST_BURN_AMOUNT, waDAI, minAmountOut, alice, orderDeadline);
```

Comment:
```suggestion
        feeBurner.burn(address(0), waDAI, TEST_BURN_AMOUNT, waDAI, amountOut, alice, orderDeadline);
```
Do you mean this? Otherwise the amount out from the query isn't used.